drivers/py/pes/pet.py

```diff
@@ -26,11 +26,13 @@
 class PET_driver(Dummy_driver):
     def __init__(self, args=None, verbose=False):
         self.error_msg = """
-The PET driver requires specification of a .json model file fitted with 
-    the PET tools, and a template file that describes the chemical makeup of 
-the structure. 
+The PET driver requires (a) a path to the results/experiment_name folder emitted by pet_train
+                        (b) a path to an ase.io.read-able file with a prototype structure
 
-Example: python driver.py -m pet -u -o model.json,template.xyz
+Other arguments to the pet.SingleStructCalculator class can be optionally
+supplied in key=value form after the required arguments.
+
+Example: python driver.py -m pet -u -o "path/to/results/name,template.xyz,device=cuda"
```

Comment:
@sirmarcel love this, maybe at a later stage we can move it into the base driver class, as I guess any PES can benefit from this flexibility